scraper/departements.py

```diff
@@ -56,3 +57,11 @@ def to_departement_number(insee_code: str) -> str:
     else:
         raise ValueError(f'Code INSEE absent de la base des codes INSEE : {insee_code}')
 
+
+def get_city(address: str) -> str:
+    cp=re.match('^.*(?P<zipcode>\d{5}).*$', address).groupdict()['zipcode']
```

Comment:
Plus simplement, fais : 
```suggestion
    return re.search('(?<= \d{5} )(?P<com_nom>.*)\s*$', address).groupdict()['com_nom']
```
Concretement, ça va recupérer tout ce qui est après le code postal jusqu'à la fin de la chaîne de caractères, trimé ; en théorie, ça correspond à la commune et ça correspond aussi à ce que tu faisais en une ligne.